Makefile

```diff
@@ -75,19 +75,19 @@ $(BUILD)/$(VER): $(SOURCE)
 	mkdir -p $(BUILD)
 	git log --pretty=format:'%H' -n 1 > $(BUILD)/$(VER)
 
-# Rule to build the un-minified library
+# Rule to build the un-minified library and convert es6 code to es5
```

Comment:
I don't see any change to the recipe for un-minified code. I would expect that it needs to be compiled here by babel.

Reply:
I missed the line. I have added it back. In addition if we don't plan on having Node installed then we might need to get rid of the `jshint` library too. As it is also run when we build the code.

Just having Node as part of our build process would be great. New `es6` features are pretty useful and reduces effort spent in handling scope and parameters.